torch/csrc/jit/codegen/cuda/scheduler/transpose.cpp

```diff
@@ -109,6 +109,9 @@ class DomainMap : public pointwise_utils::DomainMap {
     decltype(input_tvs)* tv_filtered_groups[2] = {&output_tvs, &input_tvs};
     for (auto tv_filtered_group : tv_filtered_groups) {
       for (auto tv : *tv_filtered_group) {
+        if (tv->isFusionInput() && tv->uses().empty()) {
+          continue;
+        }
```

Comment:
For example, if I have
```C++
tv0 = input
tv1 = randlike(tv0);
```
Now with the new nullary `RNGOp`, that above DAG will become disconnected. We don't want the transpose scheduler to consider `tv0` as a group. It is never used, so should be left alone and never touched.

Reply:
Haven't followed the nullary RNGop PR, but tv0 is appeared to be used by tv1. Why no use connection?

Reply:
Because after the nullary RNGOp PR, `randlike(x)` is just a nullary op `rand(x.shape(), x.dtype())`

Reply:
OK, I see.